<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureXMLManagementDialog</class>
 <widget class="QDialog" name="ConfigureXMLManagementDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xml management</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>Creation</string>
     </property>
     <property name="margin">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QCheckBox" name="chkInsProlog">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Automatically insert prolog with encoding:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboEncoding">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cbInsertFormattingMetadata">
     <property name="text">
      <string>Automatically insert formatting metadata processing instruction</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Metadata</string>
     </property>
     <property name="margin">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkMetadata">
     <property name="text">
      <string>Automatically update metadata as processing instruction if found when saving data</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="margin">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>XML &amp;indentation on save</string>
       </property>
       <property name="buddy">
        <cstring>xmlIndent</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="xmlIndent">
       <property name="maximum">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkNoIndent">
       <property name="text">
        <string>do not indent at all</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdIndentReset">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Resets the xml indentation value.</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Attributes line length:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QRadioButton" name="attrNoIndendation">
          <property name="text">
           <string>No limit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cmdPredefinedAttributes">
          <property name="text">
           <string>Predefined</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="attrNewLineAt">
          <property name="text">
           <string>New line at</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="attrCharacters">
          <property name="frame">
           <bool>true</bool>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>characters</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkSortAttributes">
     <property name="text">
      <string>Sort alphabetically the attributes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbProcessFormattingMetadata">
     <property name="text">
      <string>Process formatting metadata</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>Anonymization</string>
     </property>
     <property name="margin">
      <number>16</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cbAnonLegacyAlg">
     <property name="text">
      <string>Use legacy algorithm</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkInsProlog</tabstop>
  <tabstop>comboEncoding</tabstop>
  <tabstop>chkMetadata</tabstop>
  <tabstop>xmlIndent</tabstop>
  <tabstop>chkNoIndent</tabstop>
  <tabstop>cmdIndentReset</tabstop>
  <tabstop>attrNoIndendation</tabstop>
  <tabstop>attrNewLineAt</tabstop>
  <tabstop>attrCharacters</tabstop>
  <tabstop>cmdPredefinedAttributes</tabstop>
  <tabstop>chkSortAttributes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
